### PR TITLE
doc: update installation instructions re wheels

### PR DESCRIPTION
**Pull request by kgriffs**

### CODE REVIEW

**Review by vytas7 — APPROVED** [review]
:+1:

File: README.rst [hunk 1]
```diff
@@ -225,18 +225,23 @@ CPython
...
+Or, to install the latest beta or release candidate, if any:
+
+.. code:: bash
+
+    $ pip install --pre falcon
+
+In order to provide an extra speed boost, Falcon can compile itself with
+Cython. Wheels containing pre-compiled binaries are available from PyPI for
+several common platforms. However, if a wheel for your platform of choice is not
+available, you can choose to stick with the pure python wheel, or use the
```
Comment (CaselIT) [inline README.rst:243]: > pure python wheel

to be pedantic, it's a source distribution. We don't have pure python wheels

File: docs/user/install.rst [hunk 2]
```diff
@@ -25,18 +25,23 @@ CPython
...
+Or, to install the latest beta or release candidate, if any:
+
+.. code:: bash
+
+    $ pip install --pre falcon
+
+In order to provide an extra speed boost, Falcon can compile itself with
+Cython. Wheels containing pre-compiled binaries are available from PyPI for
+several common platforms. However, if a wheel for your platform of choice is not
+available, you can choose to stick with the pure python wheel, or use the
```
Comment (CaselIT) [inline docs/user/install.rst:43]: same here

**Review by CaselIT — APPROVED** [review]

**Comment by CaselIT** [comment]
@kgriffs I'm fine either wording, I've approved so you can merge either way

**Review by vytas7 — APPROVED** [review]

**Review by CaselIT — APPROVED** [review]

**Pull request merged**